### DISCUSSION TITLE
[PostgreSQL] Multiple identical table names in different schemas support

### DISCUSSION THREAD

**Discussion by v41e** [directus/directus#14790] [Feature Requests]
**Context**

I have an existing DB (Postgres) with many tables with many schemas: www, us, fr, es, de
In each schema I have several tables and a "company" table with different fields for each schema.

Each schema is loaded in Directus with
```env

DB_SEARCH_PATH="public,www,us,fr,es,de"

```
But the `company` collection in Directus has all the fields on the `company` tables from each schema in the same collection.

I also tried to differentiate collections and tables using Directus CLI with `directus schema snapshot` and `directus schema apply` in order to manually set the collection name as `www_company` and the table name as `company` with schema `www` but it didn't work as expected, Directus creates a `www_company` in `public` schema

**Propositions**

- prefix `table_name` with `schema_name` (if not `public`) in order to construct unique `collection_name`
- add a field in collection creation form in order to manually set table name when creating collection, with that we can easility create a collection `www_company` and associate it to the table `www.company`

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
